README.md

```diff
@@ -1,2 +1,34 @@
-# review-your-project
-This project is to showcase the things learned at the TSE training program.
+# Hello-Microverse
+
+> This project is to showcase the things learned at the TSE training program.
+
+![screenshot](./assets/Screenshot_4.png)
```

Comment:
Well done on pointing out the broken image. However, it would be better if you use polite words rather than commanding the developer to fix the issue. I would rephrase this feedback to read: _You really did well to include the project screenshot on the readme. However, the image is not being displayed on the readme. probably because the image path is not correct. Kindly let us fix the image path and ensure that the image is displayed_